OtripleS.Web.Api.Tests.Unit/Services/GuardianServiceTests/GuardianServiceTests.Exceptions.Add.cs

```diff
@@ -0,0 +1,156 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.Guardian;
+using OtripleS.Web.Api.Models.Guardian.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.GuardianServiceTests
+{
+    public partial class GuardianServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnAddWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            DateTimeOffset dateTime = GetRandomDateTime();
+            Guardian randomGuardian = CreateRandomGuardian(dateTime);
+            Guardian inputGuardian = randomGuardian;
+            inputGuardian.UpdatedBy = inputGuardian.CreatedBy;
```

Comment:
What's the point of this line?

Reply:
@viralpandya ,

Am waiting for Delete & post method of Guardian service to complete Guardiancontroller.

Reply:
> What's the point of this line?

there is a testcase that if createdby and updated by are not same while adding new guardian, then it should fail on validation.

Reply:
> @viralpandya ,
> 
> Am waiting for Delete & post method of Guardian service to complete Guardiancontroller.

Updating it. Thank you for your patience!